client/src/components/DebouncedTextInput/DebouncedTextInput.stories.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+
+import { DebouncedTextInput } from "./DebouncedTextInput";
+
+export default {
+  title: "Input/DebouncedTextInput",
+  component: DebouncedTextInput,
+};
+
+const Template = (args) => {
+  const [text, setText] = useState("");
+
+  function updateCallback(value) {
+    setText(value);
+    console.log("Text has been changed to " + value);
+  }
+
+  return (
+    <>
+      <DebouncedTextInput {...args} updateCallback={updateCallback} />
+      <div>{text}</div>
```

Comment:
Maybe label/style this text in a way that makes it clear it's not an inherent part of the component, just added to the story for demonstration.

Reply:
Done!